oak/common/app_config.bzl

```diff
@@ -20,7 +20,7 @@ def serialized_config(name, textproto, modules):
     """Serializes an Oak application configuration in a binary file.
 
     Args:
-        name: Name of the generated binary file.
+        name: Name of the generated binary file (the output file will have a `.bin` extension, e.g. `name.bin`).
```

Comment:
Please add an explicit comment before the args section. If someone is looking for the name of the file generated by this rule, they shouldn't have to look at the args section.

See https://docs.bazel.build/versions/master/be/c-cpp.html#cc_binary_implicit_outputs

Reply:
Done